controllers/AuthControllers.js

```diff
@@ -0,0 +1,74 @@
+const User = require("../models/User");
+const bcrypt = require("bcrypt");
+const jwt = require("jsonwebtoken");
+const saltRounds = 11;
+const secret = process.env.SECRET || "Four for you Glen Coco, you go Glen Coco! And none for Gretchen Weiners. Bye.";
```

Comment:
I'm not sure if this line is boiler plate or not, but if you added this. You get a million gold stars. My dog is named "Glen Coco".

Reply:
Not boilerplate but Mean Girls quote =)